src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -302,8 +347,7 @@
                                 <StackPanel Grid.Column="0"
                                             Style="{StaticResource StackPanelInExpanderStyle}">
                                     <StackPanel Orientation="Horizontal">
-                                        <TextBlock Style="{StaticResource SettingsPageItemHeaderStyle}"
-                                                   Text="{TemplateBinding Header}" />
+                                        <ContentPresenter Content="{TemplateBinding Header}" />

```

Comment:
Ugh, we have to add that whole new Style just because we can't style the text content inside a content presenter, don't we